File: bench.py
import timeit

import simplejson
import json
import ujson

def bench(binary):
  simplejson.dumps(simplejson.loads(binary))

def bench_json(binary):
  json.dumps(json.loads(binary))

def bench_ujson(binary):
  ujson.dumps(ujson.loads(binary))

if __name__ == '__main__':
  print('simplejson: ' + simplejson.__version__)
  binary = open('tokoroten.json', 'rb').read()
  print timeit.Timer("bench(binary)", "from bench import bench; binary = open('tokoroten.json', 'rb').read()").timeit(100) * 1000 * 1000

  print('json: ' + json.__version__)
  print timeit.Timer("bench_json(binary)", "from bench import bench_json; binary = open('tokoroten.json', 'rb').read()").timeit(100) * 1000 * 1000

  print('ujson: ' + ujson.__version__)
  print timeit.Timer("bench_ujson(binary)", "from bench import bench_ujson; binary = open('tokoroten.json', 'rb').read()").timeit(100) * 1000 * 1000

File: bench.php
<?php
function bench($binary) {
    json_encode(json_decode($binary));
}
$binary = file_get_contents('tokoroten.json');
$n = 100;
$t = microtime(true);
for ($i = 0; $i < $n; ++$i) {
    bench($binary);
}
echo (microtime(true) - $t) * 1000 * 1000 . PHP_EOL;
